app/controllers/admin/calendar_controller.rb

```diff
@@ -1,5 +1,25 @@
 class Admin::CalendarController < Admin::AdminController
   def index
     skip_authorization
+
+    @division = current_division
+    @loans = current_division.loans
+    @events = []
+
+    @loans.each do |loan|
+      self.prepare_event(loan.calendar_start_event)
```

Comment:
You should almost never need to use `self` in ruby. Please try to get away from this habit.